### ISSUE TITLE
[Errors] Errors use aliased type name instead of the alias name

### ISSUE THREAD

**Issue by mhegazy** [microsoft/TypeScript#94]
``` TypeScript
==== tests / cases / compiler / externalModuleExportingGenericClass_file0.ts
class C<T> {
    foo: T;
}
export = C;

==== tests / cases / compiler / externalModuleExportingGenericClass_file1.ts
import a = require('externalModuleExportingGenericClass_file0');
var v: a; // this should report error
```

Expected:
!!! Generic type 'a' requires 1 type argument(s).

Actual: 
!!! Generic type 'C' requires 1 type argument(s).

``` TypeScript
import file1 = require('externalModuleRefernceResolutionOrderInImportDeclaration_file1');
file1.foo();
```

Expected:
!!! Property 'foo' does not exist on value of type 'typeof file1'.

Actual: 
!!! Property 'foo' does not exist on type 'typeof "externalModuleRefernceResolutionOrderInImportDeclaration_file1"'.
